project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,752 @@
+  # Set secondary fuel boiler efficiency levels
+      - &secondary_boiler_72
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 72% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 72% AFUE
+      - &secondary_boiler_76
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 76% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 76% AFUE
+      - &secondary_boiler_80
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 80% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 80% AFUE
+      - &secondary_boiler_82
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 82% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 82% AFUE
+      - &secondary_boiler_85
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 85% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 85% AFUE
+      - &secondary_boiler_90
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 90% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 90% AFUE
+      - &secondary_boiler_95
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 95% AFUE, OAT Reset
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 95% AFUE, OAT Reset
+      - &secondary_boiler_96
+        option: HVAC Secondary Heating Efficiency|Fuel Boiler, 96% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Boiler, 96% AFUE
+
+  # Wall/Floor Furnace as secondary systems
+  # Set secondary Wall/Floor Furnace fuel type
+      - &ng_wallfloor_furnace_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Natural Gas Fuel Wall/Floor Furnace
+        apply_logic:
+          - HVAC Heating Type And Fuel|Natural Gas Fuel Wall/Floor Furnace
+      - &propane_wallfloor_furnace_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Propane Fuel Wall/Floor Furnace
+        apply_logic:
+          - HVAC Heating Type And Fuel|Propane Fuel Wall/Floor Furnace
+      - &fuel_oil_wallfloor_furnace_secondary_heating_type_fuel
+        option: HVAC Secondary Heating Type And Fuel|Fuel Oil Fuel Wall/Floor Furnace
+        apply_logic:
+          - HVAC Heating Type And Fuel|Fuel Oil Fuel Wall/Floor Furnace
+  # Set secondary fuel Wall/Floor Furnace efficiency levels
+      - &secondary_wallfloor_furnace_60
+        option: HVAC Secondary Heating Efficiency|Fuel Wall/Floor Furnace, 60% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 60% AFUE
+      - &secondary_wallfloor_furnace_68
+        option: HVAC Secondary Heating Efficiency|Fuel Wall/Floor Furnace, 68% AFUE
+        apply_logic:
+          - HVAC Heating Efficiency|Fuel Wall/Floor Furnace, 68% AFUE
+
+#### ducted furnace ####
+  # - upgrade_name: Dual-Fuel Natural Gas Furnace 60% AFUE
```

Comment:
I think a general comment cleanup would be helpful. The "- upgrade name" was confusing.